testing/sdk_install.py

```diff
@@ -13,7 +14,7 @@
 import sdk_plan
 import sdk_utils
 
-from retrying import retry
+log = logging.getLogger(__name__)
```

Comment:
Actually, this retry is already import *above*. 
https://github.com/mesosphere/dcos-commons/pull/1383/files#diff-a4edd5b45d2ef4fda8fd5a8e90348736R11

This  a good example of why we should be using a style checker in this repo... They have a good purpose. But that can be done later; there is no point in cleaning up stuff we might just toss out and we have some other things to take care of first.